raiden/network/pathfinding.py

```diff
@@ -179,6 +180,16 @@ def configure_pfs_or_exit(
         )
         click.secho(msg)
         log.info("Using PFS", pfs_info=pathfinding_service_info)
+        pfs_token_network_address = pathfinding_service_info["network_info"]["registry_address"]
+        if pfs_token_network_address != token_network_registry_address:
```

Comment:
can you use `is_same_address`  here, it's never clear to me which addresses are encoded in which way. (Something that we need to improve, but not in this PR)

Reply:
good point, changed it